lisa/sut_orchestrator/qemu/platform.py

```diff
@@ -68,7 +68,7 @@ def _configure_nodes(self, environment: Environment, log: Logger) -> None:
         environment_context = get_environment_context(environment)
 
         # Generate a random name for the VMs.
-        test_suffix = "".join(random.choice(string.ascii_uppercase) for i in range(5))
+        test_suffix = "".join(random.choice(string.ascii_uppercase) for _ in range(5))
```

Comment:
Out of interest, was something complaining about this? For me, this passed `make check`.

Reply:
The error is about unused variable `i`. It's from pylance, which is the new Python language server of MS. But we're using mypy in the build and UT, so there is no error in make check.